server/pbenchinacan/load-canned-data.sh

```diff
@@ -31,14 +31,9 @@ then
   exit 2
 fi
 
-if [[ $# == 1 ]]
-then
-  token=$1
-else
-  username=${1:-pbench_user}
-  password=${2:-pbench}
-  token=$(pbench-generate-token --username $username --password $password)
-fi
+# Get the Pbench authentication token
+token_location=$1
+token=$(< ${token_location})
```

Comment:
Given that `token_location` is not used except for the line immediately following its definition, let's skip defining it.

Also, given that the token string is a required parameter and that script will behave in weird ways if it's omitted, you should add a check which exits with an error if it's not provided.